client/my-sites/checkout/src/components/jetpack-akismet-checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -140,55 +140,52 @@ const JetpackAkismetCheckoutSidebarPlanUpsell: FC = () => {
 		{ strong: <strong /> }
 	);
 
+	const yearOnePrice = formatCurrency(
+		isComparisonWithIntroOffer ? currentVariant.priceInteger : currentVariant.priceBeforeDiscounts,
+		currentVariant.currency,
+		currencyConfig
+	);
+
+	const yearTwoPrice = formatCurrency(
+		currentVariant.priceBeforeDiscounts,
+		currentVariant.currency,
+		currencyConfig
+	);
+
+	const twoYearTotal = formatCurrency(
+		currentVariant.priceBeforeDiscounts +
+			( isComparisonWithIntroOffer
+				? currentVariant.priceInteger
+				: currentVariant.priceBeforeDiscounts ),
+		currentVariant.currency,
+		currencyConfig
+	);
+
+	const twoYearTotalBiennial = formatCurrency(
+		biennialVariant.priceInteger,
+		biennialVariant.currency,
+		currencyConfig
+	);
+
+	// We don't want to call out the two year plan if it doesn't save them money
+	if ( twoYearTotal >= twoYearTotalBiennial ) {
```

Comment:
Will this work since `twoYearTotal` and `twoYearTotalBiennial` are currency formatted strings? Maybe you should compare numbers instead.

Reply:
The ">=" will transform them into numbers so it does work this way, however I think it is still better to compare the numbers directly instead of using JavaScript weirdness 😅 , I'll go ahead and update this, thank you!